src/dhtproto/client/mixins/NeoSupport.d

```diff
@@ -605,6 +606,59 @@ template NeoSupport ( )
 
         mixin TaskBlockingCore!();
 
+        /***********************************************************************
+
+            Suspends the current Task until a connection has been established to
+            the node on localhost listening on the specified port and its
+            hash-range queried.
+
+            Params:
+                port = port of node to connect to
+
+        ***********************************************************************/
+
+        public void connect ( ushort port )
+        {
+            this.outer.neo.addNode("127.0.0.1", port);
```

Comment:
I find usefulness of this method questionable - I think it's better to always to pass address and port to `connect` method, to make it clear where the client connects to.

Reply:
It's not _dangerously_ lazy, but it's a bit mysterious to me if you write `connect(port)` vs `connect(address, port)`.